src/briefcase/platforms/linux/appimage.py

```diff
@@ -120,21 +125,48 @@ def verify_host(self):
 
 
 class LinuxAppImageCreateCommand(
-    LinuxAppImageMixin, LocalRequirementsMixin, CreateCommand
+    LinuxAppImageMixin,
+    LocalRequirementsMixin,
+    CreateCommand,
 ):
     description = "Create and populate a Linux AppImage."
 
-    def support_package_filename(self, support_revision):
-        """The query arguments to use in a support package query request."""
-        return f"Python-{self.python_version_tag}-linux-{self.tools.host_arch}-support.b{support_revision}.tar.gz"
+    def finalize_app_config(self, app: AppConfig):
+        """If we're *not* using Docker, verify that we're actually on Linux."""
+        if not self.use_docker:
+            self.logger.info(
+                """\
+*************************************************************************
+** WARNING: Building a Local AppImage!                                 **
+*************************************************************************
+
+    You are building an AppImage outside Docker. The resulting AppImage
+    will work, but will not be as portable as a Docker-based AppImage.
+    Any `manylinux` setting will be ignored.
+
+*************************************************************************
+"""
+            )
 
-    def support_package_url(self, support_revision):
-        """The URL of the support package to use for apps of this type."""
-        return (
-            "https://briefcase-support.s3.amazonaws.com/"
-            f"python/{self.python_version_tag}/{self.platform}/{self.tools.host_arch}/"
-            + self.support_package_filename(support_revision)
-        )
+    def output_format_template_context(self, app: AppConfig):
+        context = super().output_format_template_context(app)
+        # Add the manylinux tag to the template context.
+        try:
+            context["manylinux_tag"] = f"{app.manylinux}_{self.tools.host_arch}"
```

Comment:
> users will regularly see entire rebuilds of their Docker image.

Actually, I guess this would only be true if something re-pulled the `latest` tag....which is possible but won't just happen on its own. The reproducibility argument probably still stands, though, for environments like CI, for instance.

Reply:
I guess it can't hurt to expose the tag as a configurable option - I'll add that in.